airqo-platform/AirQo-frontend
# Update: analytics data download frontend

- What does this PR do?
This PR allows users to download data in `CSV` or `JSON` format
- What JIRA task is related to this PR?
JIRA card is [here](https://airqoteam.atlassian.net/browse/NLYTCS-104)
- How do I test out this PR?
Git clone this repo `cd AirQo-frontend/analytics`. Then run `npm install` to get all the necessary dependencies. Afterwards, run `npm run local` to test out the system. Ensure that the backend associated with this PR is running simultaneously.
